action.yml

```diff
@@ -63,6 +63,8 @@ outputs:
     description: 'Name of the image built'
   tags:
     description: 'List of the tags that were created, separated by spaces'
+  image-with-tag:
```

Comment:
let's use snake case https://github.com/redhat-actions/common/wiki/Naming-guidelines#input-and-output-names

Reply:
nevermind, everything in this action already uses kebab case.